README.md

```diff
@@ -169,11 +169,10 @@ If a `save-artifacts` script exists, a prior image already exists, and the `--in
 
 ## Dependencies
 
-1. [Docker](http://www.docker.io) >= 1.6
-1. [Go](http://golang.org/) >= 1.4
+1. [docker](https://github.com/docker/docker) >= 1.6
```

Comment:
Okay. I was updating it because `docker.io` is outdated. How about: https://www.docker.com/products/docker-engine ? Or should we just link to the main docker site, https://docker.com ?